### PR TITLE
Create and Merge snbhowmik.json

### PR DESCRIPTION
**Pull request by snbhowmik**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://snbhowmik.netlify.app

### CODE REVIEW

File: domains/snbhowmik.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "description": "A Portfolio site for my Academic Career",
+    "repo": "https://github.com/snbhowmik/portfolio",
+    "owner": {
+        "username": "snbhowmik",
+        "linkedin": "https://www.linkedin.com/in/snbhowmik/",
+    },
+    "record": {
+        "A": [
+            "75.2.60.5"
```
Comment (CuteDog5695) [inline domains/snbhowmik.json:10]: Please give your site preview.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/snbhowmik.json (hunk 1 above)
Comment (snbhowmik) [reply domains/snbhowmik.json:10]: image

File: domains/snbhowmik.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "description": "A Portfolio site for my Academic Career",
+    "repo": "https://github.com/snbhowmik/portfolio",
+    "owner": {
+        "username": "snbhowmik",
+        "linkedin": "https://www.linkedin.com/in/snbhowmik/",
```
Comment (CuteDog5695) [inline domains/snbhowmik.json:6]: You need to add email line, though you can leave it blank if you don't want to give your email.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/snbhowmik.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        "reddit": "https://www.reddit.com/user/snbhowmik/",
+        "email": "snbhowmikac@proton.me",
```
Comment (CuteDog5695) [inline domains/snbhowmik.json:8]:
```suggestion
        "email": "snbhowmikac@proton.me"
```

**Review by iostpa — DISMISSED** [review]

**Pull request merged**